app/src/lib/git/status.ts

```diff
@@ -75,9 +67,28 @@ export async function getStatus(repository: Repository): Promise<IStatusResult>
   for (const entry of parsePorcelainStatus(result.stdout)) {
     if (entry.kind === 'entry') {
       const status = mapStatus(entry.statusCode)
+
+      if (status.kind === 'ordinary') {
+        if (status.staged === GitStatusEntry.Added
+            && status.unstaged === GitStatusEntry.Deleted) {
+          // we can safely avoid drawing this file in the app
```

Comment:
So this is the case where a file has been staged but no longer exists on disk, is that right? So it won't really end up in the commit?